src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
```

Comment:
The border-radius property is not needed for stars, as they are images and this property won't affect them.